### ISSUE TITLE
NVDA not working with some edit controls in Firefox

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1431]
**Reported by KevanGC on 2011-03-26 07:11**
Here's an example. The funny frame is near the top of the page.
http://xltweet.com/

In Firefox, NVDA doesn't read what you type. You can't activate forms mode on it either unless you do it manually.

In IE, NVDA reads what you type, but it still has some problems.

It should act like any other edit box, shouldn't it?

**Comment by ehollig** [comment]
It appears the link reported in https://github.com/nvaccess/nvda/issues/1431#issue-116026178 no longer exists. Is there another example that can be provided that experiences the same behavior?

**Comment by jcsteh** [comment]
Closing, as the original page is gone and there have been a lot of changes in the way NVDA handles web contentEditable over the last few years.
